test_reminder.py

```diff
@@ -0,0 +1,50 @@
+from datetime import date, datetime
+
+# import pytest
+from freezegun import freeze_time
+
+from reminder import Reminder
+
+
+@freeze_time("2023-06-15")
+def test_get_monday():
+    test_date = date(2023, 6, 15)
+    assert Reminder().get_monday(test_date) == date(2023, 6, 12)
+
+
+@freeze_time("2023-06-12")
+def test_is_spring_planning():
+    assert not Reminder().is_sprint_planning()
+
+
+@freeze_time("2023-06-12")
+def test_next_planning():
+    assert Reminder().next_occurance("Sprint planning") == datetime(2023, 6, 19, 15, 30)
+
+
+@freeze_time("2023-06-12")
+def test_next_review():
+    assert Reminder().next_occurance("Sprint review") == datetime(2023, 6, 15, 14, 45)
+
+
+@freeze_time("2023-06-12")
+def test_next_retrospective():
+    assert Reminder().next_occurance("Sprint restrospective") == datetime(
+        2023, 6, 16, 9, 30
+    )
+
+
+@freeze_time("2023-06-16 15:32:00")
```

Comment:
do you need to mock the time?

Reply:
Not really, but I think it would be easier if this parameter didn't change when the tests will be made

Reply:
isn't the daily always at 9:30? what will change? I don't see any dependency from the current time